src/common/mpiclient/MpiClient.c

```diff
@@ -11,6 +11,7 @@
 #include <parson.h>
 #include <CommonUtils.h>
 #include <Logging.h>
+#include <Internal.h>
```

Comment:
Don't include this header here! Include Telemetry.h instead. Internal.h is internal for commonutils only!

This is blocking